src/main/scala/me.chuwy/otusbats/Monad.scala

```diff
@@ -5,9 +5,28 @@ trait Monad[F[_]] extends Functor[F] { self =>
 
   def point[A](a: A): F[A]
 
-  def flatten[A](fa: F[F[A]]): F[A]
+  def flatten[A](fa: F[F[A]]): F[A] = self.flatMap(fa)(a => a)
```

Comment:
```suggestion
  def flatten[A](fa: F[F[A]]): F[A] = self.flatMap(fa)(identity)
```

Мелочь конечно, но важно знать что есть такая функция, на практике встречается на удивление часто.